<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckinformationDialog</class>
 <widget class="QDialog" name="CheckinformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>498</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CheckinformationDialog{
border-image: url(:/mainbackground.jpg);
}
#CheckinformationDialog *{
border-image: url();
}</string>
  </property>
  <widget class="QLineEdit" name="bookpeopleEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 70%);</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookpeoplelabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>客户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="booktypelabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>105</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>证件类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="booktypecomboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>105</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(195, 213, 223, 50%);</string>
   </property>
   <item>
    <property name="text">
     <string>身份证</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>户口本</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>驾照</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>护照</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="bookidlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>证件号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookidEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 70%);</string>
   </property>
  </widget>
  <widget class="QLabel" name="telephonelabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>215</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephoneEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>215</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 70%);</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookstatelabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预定状态</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bookstatecomboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(195, 213, 223, 50%);</string>
   </property>
   <item>
    <property name="text">
     <string>预定</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>入住</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgba(195, 213, 223, 70%); 
color: black; }</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgba(195, 213, 223, 70%); 
color: black; }</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>325</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 70%);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>325</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>入住天数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
